ad-blocker-update.rst

```diff
@@ -0,0 +1,84 @@
+will see our ads again.
+The **impact to our ad views, clicks, and revenue was immediate**.
+
+.. _acceptable ads list: https://acceptableads.com/
+
+
+Data on our inclusion in the Acceptable Ads list
+------------------------------------------------
+
+.. figure:: img/2018-readthedocs-adblocker-update.png
+   :alt: Impact of ad blocking and acceptable ads at Read the Docs
+   :width: 100%
+
+32% of Read the Docs visitors run an ad blocker
+    Estimates around the web vary regarding what percentage of people
+    run ad blockers and it varies heavily by industry.
+    We discussed this figure a bit in our :doc:`previous post<ads-and-adblocking>`.
+
+It took a month from application to inclusion in the list
+    We discovered about our inclusion in the `EasyList`_ ad blocker list
+    within a couple days and immediately applied to the Acceptable Ads program.
+    No changes were required of us to meet the `Acceptable Ads criteria`_
+    and we also qualified for `Acceptable Ads without third-party tracking`_.
+
+    While Eyeo, the company behind AdBlock Plus,
+    derives `most of their revenue`_ from the Acceptable Ads initiative
+    by charging large entities,
+    Read the Docs **did not** pay for inclusion on the list.
+
+Over half of those running an ad blocker allow acceptable ads
+    Given our more tech-savvy and privacy-focused audience,
+    we didn't know what to expect.
+    Not every ad blocker allows acceptable ads by default
+    and even then users can choose to turn them off.
+
+People running ad blockers DO click on ads
+    We can definitively say this is true of our audience.
+    Our click-through rate did not change based on ad blocking or
+    our inclusion in the acceptable ads list at all.
+
+Of the ad views and clicks blocked by ad blockers,
+inclusion in the Acceptable Ads list meant that 55-60%
+of our ad-blocking visitors choose to allow our ads.
+This directly translated into revenue toward our sustainability.
+
+
+.. _EasyList: https://easylist.to/
+.. _Acceptable Ads criteria: https://acceptableads.com/en/about/criteria
+.. _Acceptable Ads without third-party tracking : https://adblockplus.org/en/acceptable-ads#privacy-friendly-acceptable-ads
+.. _most of their revenue: https://adblockplus.org/about#monetization
+
+
+Building our Ethical Ads
+------------------------
+
+The Acceptable Ads list is definitely a step in the right direction for us.
+We share the goal of unintrusive advertising that respects our audience.
+
+People understand that advertising funds much of the web.
+As ads got larger, more annoying,
+and ad networks stored more information on millions of people, however,
+many people finally decided to install ad blockers.
+This is the `largest boycott in human history`_ and blaming users isn't the answer.
+
+Based on our experience with ad blocking and the Acceptable Ads list,
+it is clear that visitors will allow advertising that doesn't disrupt user experience
+and doesn't stalk people.
```

Comment:
stalk is maybe a bit heavy here. It feels a bit out of place w/ the tone of the rest of the article.

I think we also need a bit more of a closing along the lines of :

We are glad to have made some progress towards regaining traffic that was lost. We are continuing on our experiment to fund open source infrastructure in a way that is a win/win for all parties, and we are glad you're following along with us.

Reply:
I think your point about stalking is fair. I agree with the closing line. The only thing I want to avoid is calling traffic "lost". I know some people in the ad blocking world don't like that.